mkdocs.yml

```diff
@@ -88,6 +88,7 @@ nav:
       - 'Configuration-Guide.md'
       - "Fine-Tuning.md"
       - "LoRA.md"
+      - "examples/Fine-Tuning-Function-Calling.md"
```

Comment:
should we maybe break out an examples section? with Fine-Tuning, LoRA, and this?

Reply:
Yes, or `Tutorial` that you added would be great for this.